.github/workflows/build.yaml

```diff
@@ -4,6 +4,9 @@ on:
     pull_request: ~
     push: ~
 
+env:
+    SYMFONY_DEPRECATIONS_HELPER: max[self]=0
```

Comment:
I would not add this globally. The existing behavior is that other jobs report *all* deprecations as being an issue, and that's the goal. We want the CI to report direct deprecations (to avoid our users complaining that KnpMenu triggers deprecations when using it because our CI would not have warned us).

The `-prefer-lowest` job uses a special config restricting only our own deprecations and not direct or indirect ones, because we cannot expect the lowest dependency version to be deprecation-free.